### DISCUSSION TITLE
Error 500 when access page admin

### DISCUSSION THREAD

**Discussion by cds-web-system** [microweber/microweber#929] [General]
Hi,

Afater installation  microweber v1.3.0 in my server(hestiacp) php-fpm 8.1, when i access page,  mydomain.br, all ok, page load success, but when access mydomain.br/admin to create page, show page 500 Erro Server.

Debug:
file_exists(): File name is longer than the maximum allowed path length on this platform (4096): /home/cds/web/cds.tec.br/public_html/resources/views/&lt;!DOCTYPE html&gt;
&lt;html lang=&quot;en-US&quot; dir=&quot;ltr&quot;&gt;

ErrorException
in [/home/cds/web/cds.tec.br/public_html/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php ](https://www.cds.tec.br/admin/)(line 28)
     * @param  string  $path     * @return bool     */    public function exists($path)    {        return file_exists($path);    }    /**     * Determine if a file or directory is missing.     *
in [/home/cds/web/cds.tec.br/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php ](https://www.cds.tec.br/admin/)-> handleError (line 228)
     * @return callable     */    protected function forwardsTo($method)    {        return fn (...$arguments) => static::$app            ? $this->{$method}(...$arguments)            : false;    }    /**     * Determine if the error level is a deprecation.
HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}()

**Comment by cds-web-system** [comment]
I installed the v1.2.21 it's ok

**Comment by peter-mw** [comment]
fixed in version 1.3.1
